src/Quill.php

```diff
@@ -391,10 +426,14 @@ protected function prepareOptions(): void
                 $this->addConfig('formats', $this->formats);
             }
 
-            if ($this->readOnly !== null && (bool) $this->readOnly) {
+            if ($this->readOnly !== null && (bool)$this->readOnly) {
                 $this->addConfig('readOnly', true);
             }
 
+            if ($this->smartBreak !== null && (bool)$this->smartBreak) {
```

Comment:
I think we can skip this altogether with `smartBreak` property and make it work through configuration like everything else.

Also please add

```
if (isset($this->configuration['modules']['smart-breaker'])) {
    $this->setSmartBreak(true);
}
```
in the first `if` section of this method.